src/support/serde.rs

```diff
@@ -209,12 +184,26 @@ mod tests {
             });
             proptest!(|(value: Bits<BITS, LIMBS>)| {
                 let serialized = serde_json::to_string(&value).unwrap();
-                let deserialized = serde_json::from_str(&serialized[..]).unwrap();
+                let deserialized = serde_json::from_str(&serialized).unwrap();
                 assert_eq!(value, deserialized);
             });
         });
     }
 
+    #[test]
+    fn test_human_readable_de() {
+        let jason = r#"[
+            1,
+            "0x1",
+            "0o1",
+            "0b1"
```

Comment:
test cases for prefix without body `"0x", "0o", "0b"`

Reply:
those and the empty string "" return 0, is this expected?

Reply:
It is expected based on the behavior of `from_str_radix` and `from_base_be`, yes